grype/presenter/json/presenter_test.go

```diff
@@ -188,7 +188,7 @@ func TestJsonImgsPresenter(t *testing.T) {
 	}
 
 	// TODO: add me back in when there is a JSON schema
-	// validateAgainstV1Schema(t, string(actual))
+	// validateAgainstdbSchema(t, string(actual))
```

Comment:
nit: I'd suggest capitalizing at least the "D" in "db" to make it more identifiable as a distinct word from "Against"

Reply:
I blame my search and replace regex :D will fix

Reply:
This has been fixed... with an improved and polished regular expression